java/src/test/java/com/microsoft/edge/seleniumtools/EdgeDriverTest.java

```diff
@@ -0,0 +1,33 @@
+package com.microsoft.edge.seleniumtools;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class EdgeDriverTest {

```

Comment:
Can we add some test cases to verify that the right version of edge browser (either legacy edge or chromium edge) is started with the corresponding ms:edgeChromium setup?

Reply:
See earlier comment. This EdgeDriver is Chromium-only. I only added minimal tests to verify that Edge Chromium is launched and that ms:edgeOptions is set to true.